nevergrad/functions/functionlib.py

```diff
@@ -63,6 +63,96 @@ def _short_repr(self) -> str:
         return "Photonics"
 
 
+class PBT_no_overfitting(ExperimentFunction):
+    """Population-Based Training, also known as Lamarckism or Meta-Optimization."""
+
+    def __init__(self, names: tp.List[str], dimensions: tp.List[int], num_workers: int):
+        for name in names:
+            if name not in corefuncs.registry:
+                available = ", ".join(sorted(corefuncs.registry))
+                raise ValueError(f'Unknown core function "{name}" in PBT. Available names are:\n-----\n{available}')
+        self._funcs = [corefuncs.registry[name] for name in names]
+        self._optima = [np.random.normal(size=d) for d in dimensions]
+        assert len(names) == len(dimensions)
+        self._dimension = len(names)
```

Comment:
you should use another name, dimension has a different meaning

Reply:
Done!